src/plots/smith/index.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+var getSubplotCalcData = require('../get_data').getSubplotCalcData;
+var counterRegex = require('../../lib').counterRegex;
+
+var createSmith = require('./smith');
+var constants = require('./constants');
+
+var attr = constants.attr;
+var name = constants.name;
+var counter = counterRegex(name);
+
+var attributes = {};
+attributes[attr] = {
+    valType: 'subplotid',
+    dflt: name,
+    editType: 'calc',
+    description: [
+        'Sets a reference between this trace\'s data coordinates and',
+        'a polar subplot.',
+        'If *polar* (the default value), the data refer to `layout.polar`.',
+        'If *polar2*, the data refer to `layout.polar2`, and so on.'
```

Comment:
Please update the description above.

Reply:
See 03c78eb6f342da6e5bb2ef488b6ffc683f17c275.